tests/test_config.py

```diff
@@ -26,17 +26,20 @@
 
 
 @pytest.mark.parametrize('ssl', [True, False])
-@pytest.mark.parametrize('roles', ['', 'one two three'])
 @pytest.mark.parametrize('protocol', ['http', 'https'])
-def test_get_config(ssl, protocol, roles):
+def test_get_config(ssl, protocol):
     """If a base url is specified in the environment, we use it."""
     with mock.patch.object(config, '_CONFIG', None):
         with mock.patch.object(os, 'environ', dict(os.environ)):
             token = uuid4()
+            os.environ = {}
```

Comment:
I think instead of doing this the `os.environ` patch above should be used.

Reply:
Yeah I think I maybe confused myself. I will look at this again.